loan-payoff/REPWRITERSPECS/BANNO.LOAN.PAYOFF.V1.POW

```diff
@@ -633,11 +647,31 @@ PROCEDURE READCONFIGFILESETTINGS
           TERMSLASTLINEUSED=TERMSLINESCOUNT
         END
 
-      [ Print config info]
+[* Print config info. This is a hidden parameter setting which can override the default Print
+** config info setting. Use this "PC:" parameter setting in the CFG Letter file with "ON"
+** or "OFF" to force the Print config info setting to be other than the default value of being on for
+** the first [PRINTCONFIGINFODAYS] days after the most recent program modification date or
+** program installation date, whichever is more recent.
+*]
        IF SEGMENT(LFLINE,1,3)="PC:" THEN
         DO
-         IF CHARACTERSEARCH("1YESTRUE",UPPERCASE(SEGMENT(LFLINE,4,LENGTH(LFLINE))))>0 THEN
-          PRINTCONFIGINFO=TRUE
+         TMPCHR=UPPERCASE(SEGMENT(LFLINE,4,LENGTH(LFLINE)))
+         CALL TMPCHRNLTS
```

Comment:
Technically I think you could just get away with a call to TMPCHRNLS instead of TMPCHRNLTS since the use of LENGTH(LFLINE) in the SEGMENT statement in the previous line would have already trimmed off any possible leading spaces.  Line 672 as well.

Reply:
Thanks. I've made a new commit with your suggested changes